### ISSUE TITLE
name "pages/ads/index.tsx" on pages routing is not working

### ISSUE THREAD

**Issue by topzdev** [vercel/next.js#27526]
### What example does this report relate to?

routing

### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

v12.18.1

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying your application?

aws

### Describe the Bug

I tried to name my route as "pages/ads/index.tsx" I got no error but it won't render anything on dom but the head meta tags have been generated. 

It's a weird bug because I tried to name my route "pages/advertise/index.tsx" and it worked.

### Expected Behavior

Will render elements on DOM even the route name is "ads/index.tsx"

### with 'pages/ads/index.tsx' route naming:

![image](https://user-images.githubusercontent.com/44505743/127132353-553fe4ee-cf05-4814-a8d2-889e56de1f44.png)

![image](https://user-images.githubusercontent.com/44505743/127133762-82db4254-ece0-43fa-8959-74d42fd061a1.png)

---

### with 'pages/advetise/index.tsx' route naming and its working...:

![image](https://user-images.githubusercontent.com/44505743/127132776-8c1039f4-479c-49e7-911a-83b6291e4cf4.png)

![image](https://user-images.githubusercontent.com/44505743/127133643-aaff70e5-df0a-43ca-8dce-bf0aa3b7882a.png)

---

### Sample code snippet in pages/ads/index.tsx
```
import React from "react";
import Button from "@/components/button/Button";
import Container from "@/components/layout/Container";
import SeoHead from "@/components/seo/SeoHead";
import { advertiseRequestPageRoute } from "@/configs/routes";

interface AdsIndexPageProps {}

const AdsIndexPage: React.FC<AdsIndexPageProps> = ({}) => {
  return (
    <Container>
      <SeoHead title="Advertise with us!" titleTemplate="title-site" />
      <h1>Advetise with us page</h1>
      <Button href={advertiseRequestPageRoute()}>
        Advertisement Form Page
      </Button>
    </Container>
  );
};

export default AdsIndexPage;
```

**Comment by awareness481** [comment]
This is likely getting blocked by your adblock.

**Comment by topzdev** [comment]
Ohh, I tried to turn off Adblock from brave and it works! thank you.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
